components/thumbnail/basic/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+lib
+node_modules
```

Comment:
I think this is something enablers/code owners should talk about and solve properly in all components, for now this file is being autogenerated by sui-studio.

I can create an sui-studio issue if it's ok for you.